lib/IRGen/GenClass.cpp

```diff
@@ -245,6 +249,10 @@ namespace {
         }
 
         if (superclass->hasClangNode()) {
+          // Perform fragile layout if the class has @objc ancestry.
```

Comment:
This comment's out of date since we ended up deciding not to conditionalize on that.

Reply:
The comment is still valid. Note that we only set CompletelyFragileLayout if the superclass has a Clang node.

Reply:
Oh. That's not sufficient; you could be the grandchild of an imported class. I'm…sorry for not catching that sooner.

Reply:
That's fine. We walk all superclasses up to the root here.